src/components/MainContent/MainContent.css

```diff
@@ -0,0 +1,24 @@
+.MainContent {
+  grid-row: 2/3;
+  background-color: #282c34;
+  flex-direction: column;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  font-size: 16px;
+  color: white;
+}
+
+button {
```

Comment:
this style will be applied for all <button /> elements. It's better to add some className to 'GO BACK TO HOME' button to specify styles for only this button (the same for future buttons)